Gems/Atom/RHI/Code/Include/Atom/RHI/RHISystemInterface.h

```diff
@@ -65,6 +65,10 @@ namespace AZ::RHI
         virtual void QueueRayTracingShaderTableForBuild(RayTracingShaderTable* rayTracingShaderTable) = 0;
             
         virtual XRRenderingInterface* GetXRSystem() const = 0;
+
+        virtual void SetDrawListTagEnabledByDefault(DrawListTag drawListTag, bool enabled) = 0;
+
+        virtual const AZStd::vector<DrawListTag>& GetDrawListTagsDisabledByDefault() const = 0;
```

Comment:
Not a fan of adding this to RHISystem which is a global singleton. This functionality is specifically for Mesh's draw items so can this not be part of meshfeatureprocessor? What happens if you have multiple pipelines, multiple views etc? 

Maybe m_drawListTagsDisabledByDefault can live within meshfeatureprocessor and is passed to RHI when it is creating the draw packets. Meshfeatureprocessor can also ping RPI to query the RasterPassData of the active render pipeline for the disabled draw list tags. Thoughts?

Reply:
Putting it in MeshFeatureProcessor has those same problems as there's at most one instance of a FP per scene and a scene can have multiple pipelines and views. Those conflicts can't be worked around because the MeshFP creates a single DrawPacket for all render view and render pipelines of the scene, it doesn't create DrawPackets or DrawItems on a per-pipeline or per-view basis. If you have multiple pipelines with multiple passes with the same draw list tag, there will still only be one draw item, so that draw item needs to be enabled at creation or disabled.
I tried moving this into the Scene itself just to move the logic away from RHI concerns, but it doesn't work, the pass doesn't get information about the scene until after the DrawItems have been built by the MeshFP, at which points adding it's DrawListTag to the Scene's disabled list is too late.
RHI System is the best place I can think of, it's always available so the passes can immediately add their draw list tags and it's also available to the DrawPacketBuilder (as opposed to the RPI), so querying that list is easier and avoids passing arguments through multiple layers of functions.